app/javascript/components/time_table/accounting_periods/periods.js

```diff
@@ -145,10 +147,25 @@ class Periods extends React.Component {
     Api.makePostRequest({
       url: '/api/accounting_periods/recount',
       body: { user_id: this.state.userId },
-    }).then(() => {
+    }).then((response) => {
       this.setState({
+        recountJid: response.data.jid,
         recounting: true,
       });
+      this.recountingTimer = setInterval(() => this.getRecountingPeriodsState(), 1000);
```

Comment:
`getRecountingPeriodsState` is already bound to this, wrapping it in arrow function is unneded.